lua/gatekeeper.lua

```diff
@@ -133,9 +139,80 @@ struct lls_config {
 };
 
 struct gt_config {
+	unsigned int max_num_ipv4_rules;
+	unsigned int num_ipv4_tbl8s;
+	unsigned int max_num_ipv6_rules;
+	unsigned int num_ipv6_tbl8s;
```

Comment:
Once a Lua policy can take care of its LPM tables, these parameters will become useless.